lib/knock/tenants.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'uri'
+
+module Knock
+  # Methods for interacting with tenants in Knock
+  module Tenants
+    class << self
+      include Base
+      include Client
+
+      DEFAULT_PREFERENCE_SET_ID = 'default'
+
+      # Retrieves all Tenants in environment
+      #
+      # @return [Hash] Paginated list of Tenant records
+      def list
```

Comment:
We should be passing options to this endpoint for filters and page size and such?